plugins/calm-vscode/README.md

```diff
@@ -0,0 +1,74 @@
+# calm-validate README
+
+
+TODO - this needs to be populated once initial features are agreed upon.
```

Comment:
What are we thinking for initial features then:

- command to visualize instantiation and pattern
- the in-file validation errors and warnings

That should be plenty for a first release